src/diagnostic-channel-publishers/src/mysql.pub.ts

```diff
@@ -101,7 +101,7 @@ const mysqlPatchFunction: PatchFunction = function(originalMysql, originalMysqlP
 };
 
 export const mysql: IModulePatcher = {
-    versionSpecifier: ">= 2.0.0 <= 2.14.0",
+    versionSpecifier: ">= 2.0.0 <= 2.14.x",

```

Comment:
I did, the tests passed on 2.14.1 (the reason why I made this change is that I saw 2.14.1 was available when making the readme). I updated the package.json to be ^2.13.0 so it should always pull in the latest 2.x, if I understand semver correctly. I know for a fact that that change made it pull down 2.14.1 when I reinstalled.